docs: Document A/B testing for Segment Emails (#811)
mautic/user-documentation: docs/channels/emails.rst

Mautic initiates the sending of these Emails with a :doc:`/configuration/cron_jobs` - see section on Send Scheduled Broadcasts - for example, Segment Emails - for more details on this.

> .. _ab testing for segment emails:

Reviewer: ```suggestion
.. _ab testing for Segment Emails:
```

> .. vale off

Author: New A/B testing feature for Segment Emails introduced in [PR #15897](https://github.com/mautic/mautic/pull/15897). This PR adds the ability to convert Segment Emails to A/B tests with the 'Create an A/B test' card, manage variants, schedule tests, and determine winners via the mautic:email:sendwinner command.

Source: https://github.com/mautic/mautic/pull/15897

A/B testing for Segment Emails
==============================

.. vale on

A/B testing lets you split test a Segment Email against one or more variants to find the version that performs best. Mautic sends each variant to a small slice of your audience, picks a winner based on criteria you choose, then sends the winning version to everyone else.

.. note::

   You can only convert a Segment Email to an A/B test before you send it. The 'Convert to A/B test' card appears only on Segment Emails with zero sends.

.. vale off

Converting an Email to an A/B test
----------------------------------

.. vale on

#. Open a Segment Email that you haven't sent yet.
#. In the right-hand column of the Email detail view, find the 'Convert to A/B test' card and select it.

> 

Author: The 'Convert to A/B test' card and the 'Convert to A/B test email' dialog labels come from PR #15897: `messages.ini` renames `mautic.core.ab_test.create` to "Convert to A/B test", and the dialog field labels 'How to pick the winner' (`mautic.core.ab_test.form.winner`), 'Wait before picking winner' (`send_winner_delay`), and 'Test audience size' (`traffic_total_weight`) are added in the same file. Defaults (24 hours, 10%) come from `GenerateABTestType`/`ABTestController` (`DEFAULT_DELAY = 24`, `DEFAULT_AB_WEIGHT = 10`).

Source: https://github.com/mautic/mautic/pull/15897

   .. image:: https://promptless-customer-doc-assets.s3.us-east-2.amazonaws.com/docs-images/org_3A4Y7R0duvqDFH8Goz5djcVpUK3/email-ab-test-convert-card-e873976b.png
     :width: 400
     :alt: Screenshot showing the 'Convert to A/B test' card in the right-hand column of a Segment Email.

> #. In the 'Convert to A/B test email' dialog, set the initial test parameters:

Reviewer: The word 'email' here is how it appears on the UI. Wrap the whole ordered list in vale off/on statements.

   * **How to pick the winner** - the criteria Mautic uses to choose the winning variant, such as read - open - rate or click-through rate.
   * **Wait before picking winner** - how long Mautic waits, in hours, after sending the test Emails before it picks the winner. This gives Contacts time to open and click. The default is 24 hours.
   * **Test audience size** - the percentage of your Contacts that receive the test Emails, split evenly between variants. The remaining Contacts receive the winning version. The default is 10%.

   .. image:: https://promptless-customer-doc-assets.s3.us-east-2.amazonaws.com/docs-images/org_3A4Y7R0duvqDFH8Goz5djcVpUK3/email-ab-test-convert-dialog-4677cc37.png
     :width: 400

>      :alt: Screenshot showing the 'Convert to A/B test email' dialog with the winner criteria, wait time, and test audience size fields.

Reviewer: Don't start with 'Screenshot'. Refer to this guidelines: https://contribute.mautic.org/en/latest/contributing/contributing_docs.html#updating-mautic-ui-images

#. Select 'Save & Close'.

These settings only set up the test - you can change them any time before the test starts sending. After conversion, the right-hand column replaces the card with the 'A/B Test' panel, where you manage variants, settings, and sending from one place.

.. vale off

Managing variants
-----------------

.. vale on

You manage the whole test from the 'A/B Test' panel in the right-hand column. The original Email counts as the first variant, so you need at least one more - two in total - before you can schedule the test. Until you add one, the panel shows a reminder.

* To add a variant, select 'Add variant +'. Mautic creates a copy that you can edit - change the subject line, content, or design you want to test. Each variant appears in the panel tagged as an 'A/B variant' of the original Email.
* To edit a variant, select its edit icon in the panel.
* To remove a variant, select its remove icon in the panel.

> Mautic divides the test audience evenly between all published variants, including the original Email.

Reviewer: Address the Vale warning:

[Mautic.FeatureList] Is this referring to a Mautic feature? If so, use 'Active or Available' instead of 'published'.

.. vale off

Adjusting A/B test settings
---------------------------

.. vale on

To change the winner criteria, wait time, or test audience size after conversion, select the settings icon at the top of the 'A/B Test' panel. You can adjust these values any time before the test starts sending.

.. vale off

Scheduling the A/B test
-----------------------

.. vale on

> Once you have at least two variants, the panel shows a 'Schedule sending' button along with a summary of the test - for example, '10% of your contacts will receive test variants. The winner will be picked after 24 hours.'

Reviewer: Address the Vale warnings:

- [Google.Passive] In general, use active voice instead of passive voice ('be picked').
- [Google.Will] Avoid using 'will'.
- [Mautic.FeatureList] Is this referring to a Mautic feature? If so, use 'Contacts' instead of 'contacts'.

#. Select 'Schedule sending'.
#. Choose the date and time to begin sending, then confirm.

The Email moves to a 'Scheduled for future send' state, and the panel shows when sending starts. To change the timing, use the 'Reschedule sending' button.

.. note::

   For A/B test Emails, Mautic hides the 'Send' button in the header. Schedule and send the test from the 'A/B Test' panel using 'Schedule sending'.

> .. vale off

Author: The `mautic:email:sendwinner` command is introduced in PR #15897 via `SendWinnerEmailCommand` (`COMMAND_NAME = 'mautic:email:sendwinner'`, optional `--id` option). The behavior that remaining Emails are sent through the standard broadcast after the winner is determined comes from `BroadcastSubscriber::onBroadcast` A/B test logic in the same PR.

Source: https://github.com/mautic/mautic/pull/15897

   * **Test audience size** - the percentage of your Contacts that receive the test Emails, split evenly between variants. The remaining Contacts receive the winning version. The default is 10%.

   .. image:: https://promptless-customer-doc-assets.s3.us-east-2.amazonaws.com/docs-images/org_3A4Y7R0duvqDFH8Goz5djcVpUK3/email-ab-test-convert-dialog-4677cc37.png
     :width: 400
     :alt: The 'Convert to A/B test email' dialog with the winner criteria, wait time, and test audience size fields.

#. Select 'Save & Close'.

.. vale on

These settings only set up the test - you can change them any time before the test starts sending. After conversion, the right-hand column replaces the card with the 'A/B Test' panel, where you manage variants, settings, and sending from one place.

.. vale off

Managing variants
-----------------

.. vale on

You manage the whole test from the 'A/B Test' panel in the right-hand column. The original Email counts as the first variant, so you need at least one more - two in total - before you can schedule the test. Until you add one, the panel shows a reminder.

> * To add a variant, select 'Add variant +'. Mautic creates a copy that you can edit - change the subject line, content, or design you want to test. Each variant appears in the panel tagged as an 'A/B variant' of the original Email.

Reviewer: Maybe we can make intsructions here as an ordered list for easy read.

#. To edit a variant, select its edit icon in the panel.

> #. To remove a variant, select its remove icon in the panel.

Reviewer: I mean like this.

```suggestion
* To add a variant:
  
  #. Select **Add variant +**. Mautic creates a copy that you can edit.
  #. Change the subject line, content, or design you want to test. Each variant appears in the panel tagged as an 'A/B variant' of the original Email.

* To edit a variant, select its edit icon in the panel.
* To remove a variant, select its remove icon in the panel.
```

Mautic divides the test audience evenly between all Active variants, including the original Email.

.. vale off

Adjusting A/B test settings
---------------------------

.. vale on

To change the winner criteria, wait time, or test audience size after conversion, select the settings icon at the top of the 'A/B Test' panel. You can adjust these values any time before the test starts sending.

.. vale off

Scheduling the A/B test
-----------------------

.. vale on

> Once you have at least two variants, the panel shows a 'Schedule sending' button along with a summary of the test - for example, that 10% of your Contacts receive test variants and that Mautic picks the winner after 24 hours.

Reviewer: ```suggestion
Once you have at least two variants, the panel shows a **Schedule sending** button along with a summary of the test - for example, that 10% of your Contacts receive test variants and that Mautic picks the winner after 24 hours.
```

> #. Select 'Schedule sending'.

Reviewer: ```suggestion
#. Select **Schedule sending**.
```

#. Choose the date and time to begin sending, then confirm.

> The Email moves to a 'Scheduled for future send' state, and the panel shows when sending starts. To change the timing, use the 'Reschedule sending' button.

Reviewer: ```suggestion
The Email moves to a **Scheduled for future send** state, and the panel shows when sending starts. To change the timing, use the **Reschedule sending** button.
```

.. note::

>    For A/B test Emails, Mautic hides the 'Send' button in the header. Schedule and send the test from the 'A/B Test' panel using 'Schedule sending'.

Reviewer: ```suggestion
   For A/B test Emails, Mautic hides the **Send** button in the header. Schedule and send the test from the 'A/B Test' panel using **Schedule sending**.
```

> .. vale off
> 
> Sending and determining the winner
> -----------------------------------
> 
> .. vale on

Reviewer: ```suggestion
Sending and determining the winner
-----------------------------------
```

Mautic initiates the sending of these Emails with a :doc:`/configuration/cron_jobs` - see section on Send Scheduled Broadcasts - for example, Segment Emails - for more details on this.

.. _ab testing for Segment Emails:

.. vale off

A/B testing for Segment Emails
==============================

.. vale on

A/B testing lets you split test a Segment Email against one or more variants to find the version that performs best. Mautic sends each variant to a small slice of your audience, picks a winner based on criteria you choose, then sends the winning version to everyone else.

.. note::

>    You can only convert a Segment Email to an A/B test before you send it. The 'Convert to A/B test' card appears only on Segment Emails with zero sends.

Reviewer: ```suggestion
   You can only convert a Segment Email to an A/B test before you send it. The **Convert to A/B test** card appears only on Segment Emails with zero sends.
```

.. vale off

Converting an Email to an A/B test
----------------------------------

.. vale on

.. vale off

#. Open a Segment Email that you haven't sent yet.

> #. In the right-hand column of the Email detail view, find the 'Convert to A/B test' card and select it.

Reviewer: ```suggestion
#. In the right-hand column of the Email detail view, find the **Convert to A/B test** card and select it.
```

   .. image:: https://promptless-customer-doc-assets.s3.us-east-2.amazonaws.com/docs-images/org_3A4Y7R0duvqDFH8Goz5djcVpUK3/email-ab-test-convert-card-e873976b.png
     :width: 400
     :alt: The 'Convert to A/B test' card in the right-hand column of a Segment Email.

> #. In the 'Convert to A/B test email' dialog, set the initial test parameters:

Reviewer: ```suggestion
#. In the **Convert to A/B test emai** dialog, set the initial test parameters:
```

> 
> .. vale off
> 
> #. Open a Segment Email that you haven't sent yet.
> #. In the right-hand column of the Email detail view, find the **Convert to A/B test** card and select it.
> 
>    .. image:: https://promptless-customer-doc-assets.s3.us-east-2.amazonaws.com/docs-images/org_3A4Y7R0duvqDFH8Goz5djcVpUK3/email-ab-test-convert-card-e873976b.png
>      :width: 400
>      :alt: The 'Convert to A/B test' card in the right-hand column of a Segment Email.
> 
> #. In the **Convert to A/B test email** dialog, set the initial test parameters:

Reviewer: Add spaces before and after the image.

```suggestion
#. In the right-hand column of the Email detail view, find the **Convert to A/B test** card and select it.

   |

   .. image:: https://promptless-customer-doc-assets.s3.us-east-2.amazonaws.com/docs-images/org_3A4Y7R0duvqDFH8Goz5djcVpUK3/email-ab-test-convert-card-e873976b.png
      :width: 400
      :alt: The 'Convert to A/B test' card in the right-hand column of a Segment Email.
      
    |

#. In the **Convert to A/B test email** dialog, set the initial test parameters:
```

>    * **How to pick the winner** - the criteria Mautic uses to choose the winning variant, such as read - open - rate or click-through rate.

Reviewer: Write as appears on the UI.

```suggestion
   * **How to pick the winner** - the criteria Mautic uses to choose the winning variant, such as **Read rate*** or **Clickthrough rate**.
```

>    * **Wait before picking winner** - how long Mautic waits, in hours, after sending the test Emails before it picks the winner. This gives Contacts time to open and click. The default is 24 hours.

Reviewer: ```suggestion
   * **Wait before picking winner** - how long Mautic waits - in hours - after sending the test Emails before it picks the winner. This gives Contacts time to open and click. The default is 24 hours.
```

   * **Test audience size** - the percentage of your Contacts that receive the test Emails, split evenly between variants. The remaining Contacts receive the winning version. The default is 10%.

   .. image:: https://promptless-customer-doc-assets.s3.us-east-2.amazonaws.com/docs-images/org_3A4Y7R0duvqDFH8Goz5djcVpUK3/email-ab-test-convert-dialog-4677cc37.png
     :width: 400
     :alt: The 'Convert to A/B test email' dialog with the winner criteria, wait time, and test audience size fields.

> #. Select 'Save & Close'.

Reviewer: ```suggestion
#. Select **Save & Close**.
```

> #. Select **Save & Close**.

Reviewer: Fix indentation and add space to the image.

```suggestion
   * **Test audience size** - the percentage of your Contacts that receive the test Emails, split evenly between variants. The remaining Contacts receive the winning version. The default is 10%.
   
     |

     .. image:: https://promptless-customer-doc-assets.s3.us-east-2.amazonaws.com/docs-images/org_3A4Y7R0duvqDFH8Goz5djcVpUK3/email-ab-test-convert-dialog-4677cc37.png
        :width: 400
        :alt: The 'Convert to A/B test email' dialog with the winner criteria, wait time, and test audience size fields.
        
     |

#. Select **Save & Close**.
```

.. vale on

> These settings only set up the test - you can change them any time before the test starts sending. After conversion, the right-hand column replaces the card with the 'A/B Test' panel, where you manage variants, settings, and sending from one place.

Reviewer: ```suggestion
These settings only set up the test. You can change them any time before the test starts sending. After conversion, the right-hand column replaces the card with the **A/B Test** panel, where you manage variants, settings, and sending from one place.
```

.. vale off

Managing variants
-----------------

.. vale on

> You manage the whole test from the 'A/B Test' panel in the right-hand column. The original Email counts as the first variant, so you need at least one more - two in total - before you can schedule the test. Until you add one, the panel shows a reminder.

Reviewer: ```suggestion
You manage the whole test from the **A/B Test** panel in the right-hand column. The original Email counts as the first variant, so you need at least one more - two in total - before you can schedule the test. Until you add one, the panel shows a reminder.
```

* To add a variant:

  #. Select **Add variant +**. Mautic creates a copy that you can edit.
  #. Change the subject line, content, or design you want to test. Each variant appears in the panel tagged as an 'A/B variant' of the original Email.

* To edit a variant, select its edit icon in the panel.
* To remove a variant, select its remove icon in the panel.

Mautic divides the test audience evenly between all Active variants, including the original Email.

.. vale off

Adjusting A/B test settings
---------------------------

.. vale on

> To change the winner criteria, wait time, or test audience size after conversion, select the settings icon at the top of the 'A/B Test' panel. You can adjust these values any time before the test starts sending.

Reviewer: ```suggestion
To change the winner criteria, wait time, or test audience size after conversion, select the settings icon at the top of the **A/B Test** panel. You can adjust these values any time before the test starts sending.
```

.. vale off

Scheduling the A/B test
-----------------------

.. vale on

Once you have at least two variants, the panel shows a **Schedule sending** button along with a summary of the test - for example, that 10% of your Contacts receive test variants and that Mautic picks the winner after 24 hours.

#. Select **Schedule sending**.
#. Choose the date and time to begin sending, then confirm.

The Email moves to a **Scheduled for future send** state, and the panel shows when sending starts. To change the timing, use the **Reschedule sending** button.

.. note::

   For A/B test Emails, Mautic hides the **Send** button in the header. Schedule and send the test from the 'A/B Test' panel using **Schedule sending**.

.. vale off

Sending and determining the winner
-----------------------------------

.. vale on

When sending begins, Mautic sends the test Emails to the test audience and the panel shows that the test is running, along with a countdown to when Mautic picks the winner.

> After the wait time passes, the ``mautic:email:sendwinner`` command evaluates each variant against your winner criteria, marks the best performer with a 'Winner' badge in the panel, and sends the remaining Emails using the winning variant's content through the standard broadcast command. For setup details, see the :ref:`Determine A/B test winner <determine ab test winner>` section in :doc:`/configuration/cron_jobs`.

Reviewer: ```suggestion
After the wait time passes, the ``mautic:email:sendwinner`` command evaluates each variant against your winner criteria, marks the best performer with a **Winner** badge in the panel, and sends the remaining Emails using the winning variant's content through the standard broadcast command. For setup details, see the :ref:`Determine A/B test winner <determine ab test winner>` section in :doc:`/configuration/cron_jobs`.
```

.. vale off

Viewing A/B test results
------------------------

.. vale on

> Once the test is sending, the panel shows an 'A/B Test Stats' button. Select it to compare how each variant is performing and to see which variant Mautic picked as the winner.

Reviewer: ```suggestion
Once the test is sending, the panel shows an **A/B Test Stats** button. Select it to compare how each variant is performing and to see which variant Mautic picked as the winner.
```

Mautic initiates the sending of these Emails with a :doc:`/configuration/cron_jobs` - see section on Send Scheduled Broadcasts - for example, Segment Emails - for more details on this.

.. _ab testing for Segment Emails:

.. vale off

A/B testing for Segment Emails
==============================

.. vale on

A/B testing lets you split test a Segment Email against one or more variants to find the version that performs best. Mautic sends each variant to a small slice of your audience, picks a winner based on criteria you choose, then sends the winning version to everyone else.

.. note::

   You can only convert a Segment Email to an A/B test before you send it. The **Convert to A/B test** card appears only on Segment Emails with zero sends.

.. vale off

Converting an Email to an A/B test
----------------------------------

.. vale on

.. vale off

#. Open a Segment Email that you haven't sent yet.
#. In the right-hand column of the Email detail view, find the **Convert to A/B test** card and select it.

   |

   .. image:: https://promptless-customer-doc-assets.s3.us-east-2.amazonaws.com/docs-images/org_3A4Y7R0duvqDFH8Goz5djcVpUK3/email-ab-test-convert-card-e873976b.png

>       :width: 400

Reviewer: Replace the image with this:

<img width="1994" height="832" alt="Image" src="https://github.com/user-attachments/assets/9a2e848a-de24-408e-ba00-eb2adbb48dae" />

- Name the image: email_ab_test_convert_card.png
- Store this in the channels' image folder.
- Change the width to 800

      :alt: The 'Convert to A/B test' card in the right-hand column of a Segment Email.

   |

#. In the **Convert to A/B test email** dialog, set the initial test parameters:

   * **How to pick the winner** - the criteria Mautic uses to choose the winning variant, such as **Read rate** or **Clickthrough rate**.
   * **Wait before picking winner** - how long Mautic waits - in hours - after sending the test Emails before it picks the winner. This gives Contacts time to open and click. The default is 24 hours.
   * **Test audience size** - the percentage of your Contacts that receive the test Emails, split evenly between variants. The remaining Contacts receive the winning version. The default is 10%.

     |

>      .. image:: https://promptless-customer-doc-assets.s3.us-east-2.amazonaws.com/docs-images/org_3A4Y7R0duvqDFH8Goz5djcVpUK3/email-ab-test-convert-dialog-4677cc37.png

Reviewer: Store the image in the channels' image folder with name:  email_ab_test_convert_dialog.png

>         :width: 400

Reviewer: The image is too small here. Remove the width.

        :alt: The 'Convert to A/B test email' dialog with the winner criteria, wait time, and test audience size fields.

     |

#. Select **Save & Close**.

.. vale on

These settings only set up the test. You can change them any time before the test starts sending. After conversion, the right-hand column replaces the card with the **A/B Test** panel, where you manage variants, settings, and sending from one place.

.. vale off

Managing variants
-----------------

.. vale on

You manage the whole test from the **A/B Test** panel in the right-hand column. The original Email counts as the first variant, so you need at least one more - two in total - before you can schedule the test. Until you add one, the panel shows a reminder.

* To add a variant:

  #. Select **Add variant +**. Mautic creates a copy that you can edit.

>   #. Change the subject line, content, or design you want to test. Each variant appears in the panel tagged as an 'A/B variant' of the original Email.

Reviewer: In the list Emails, a variant has a symbol as the screenshot below. When users hover over the symbol, it shows 'Is A/B variant'.

Maybe we can add this image? Name it: is_ab_variant_symbol.png. Add a descriptive alt text; don't start with "screenshot of" or anything like that. Don't add width. Add a space before and after the image to give it space with the texts.

<img width="350" height="240" alt="Image" src="https://github.com/user-attachments/assets/e9fd9e96-31fc-4212-83bb-ff3fc620ca36" />

* To edit a variant, select its edit icon in the panel.

> * To remove a variant, select its remove icon in the panel.

Reviewer: Another way to edit and remove a variant is on the list of Emails, click the three dots next to the Email variant that users want to modify. Then, click 'edit' to edit a variant and 'delete' to delete it.

Mautic divides the test audience evenly between all Active variants, including the original Email.

.. vale off

Adjusting A/B test settings
---------------------------

.. vale on

> To change the winner criteria, wait time, or test audience size after conversion, select the settings icon at the top of the **A/B Test** panel. You can adjust these values any time before the test starts sending.

Reviewer: ```suggestion
To change the winner criteria, wait time, or test audience size after conversion, select the settings - gear icon - at the top of the **A/B Test** panel. You can adjust these values any time before the test starts sending.
```

.. vale off

Scheduling the A/B test
-----------------------

.. vale on

Once you have at least two variants, the panel shows a **Schedule sending** button along with a summary of the test - for example, that 10% of your Contacts receive test variants and that Mautic picks the winner after 24 hours.

> 

Reviewer: Add this image.

<img width="472" height="973" alt="Image" src="https://github.com/user-attachments/assets/a7721a22-aa29-414d-93e8-3e346aebb9b2" />

- Name it: ab_test_schedule_panel.png
- Add descriptive alt text. Don't start with "screenshot of".
- Set the height to 600

#. Select **Schedule sending**.
#. Choose the date and time to begin sending, then confirm.

The Email moves to a **Scheduled for future send** state, and the panel shows when sending starts. To change the timing, use the **Reschedule sending** button.

.. note::

   For A/B test Emails, Mautic hides the **Send** button in the header. Schedule and send the test from the 'A/B Test' panel using **Schedule sending**.

.. vale off

Sending and determining the winner

> -----------------------------------

Reviewer: Remove excess dash.

```suggestion
Sending and determining the winner
----------------------------------
```

   * **Test audience size** - the percentage of your Contacts that receive the test Emails, split evenly between variants. The remaining Contacts receive the winning version. The default is 10%.

     |

     .. image:: images/emails/email_ab_test_convert_dialog.png
        :alt: The 'Convert to A/B test email' dialog with the winner criteria, wait time, and test audience size fields.

     |

#. Select **Save & Close**.

.. vale on

These settings only set up the test. You can change them any time before the test starts sending. After conversion, the right-hand column replaces the card with the **A/B Test** panel, where you manage variants, settings, and sending from one place.

.. vale off

Managing variants
-----------------

.. vale on

You manage the whole test from the **A/B Test** panel in the right-hand column. The original Email counts as the first variant, so you need at least one more - two in total - before you can schedule the test. Until you add one, the panel shows a reminder.

* To add a variant:

  #. Select **Add variant +**. Mautic creates a copy that you can edit.

>   #. Change the subject line, content, or design you want to test. Each variant appears in the panel tagged as an 'A/B variant' of the original Email. In the list of Emails, a variant has a symbol. When you hover over the symbol, it shows 'Is A/B variant'.

Reviewer: ```suggestion
* To add a variant:

  #. Select **Add variant +**. Mautic creates a copy that you can edit.
  #. Change the subject line, content, or design you want to test.
  #. Click **Save & Close**.
  
  Each variant appears in the panel tagged as an 'A/B variant' of the original Email. In the list of Emails, a variant has a symbol. When you hover over the symbol, it shows 'Is A/B variant'.
```

     |

     .. image:: images/emails/is_ab_variant_symbol.png
        :alt: The A/B variant symbol beside an Email in the Emails list, showing the 'Is A/B variant' label on hover.

     |

* To edit a variant, select its edit icon in the panel.
* To remove a variant, select its remove icon in the panel.

> * Alternatively, edit or remove a variant from the list of Emails: select the three dots next to the Email variant you want to modify, then select 'Edit' to edit the variant or 'Delete' to delete it.

Reviewer: ```suggestion
* There are two ways to edit and remove a variant:

  #. From individual variant:
  
     #. Open the Email variant you want to modify.
     #. To edit a variant, click the **Edit** button at the top right.
     #. To remove a variant, click the down arrow button to open the **Options**, then select **Delete**.

  #. From the list of Emails:
  
     #. Select the three dots icon next to the Email variant you want to modify.
     #. Select **Edit** to edit the variant or **Delete** to remove it.
```

Scheduling the A/B test
-----------------------

.. vale on

Once you have at least two variants, the panel shows a **Schedule sending** button along with a summary of the test - for example, that 10% of your Contacts receive test variants and that Mautic picks the winner after 24 hours.

|

.. image:: images/emails/ab_test_schedule_panel.png
   :height: 600
   :alt: The A/B Test panel showing the 'Schedule sending' button and a summary of the test audience percentage and winner wait time.

|

#. Select **Schedule sending**.
#. Choose the date and time to begin sending, then confirm.

The Email moves to a **Scheduled for future send** state, and the panel shows when sending starts. To change the timing, use the **Reschedule sending** button.

.. note::

   For A/B test Emails, Mautic hides the **Send** button in the header. Schedule and send the test from the 'A/B Test' panel using **Schedule sending**.

.. vale off

Sending and determining the winner
----------------------------------

.. vale on

When sending begins, Mautic sends the test Emails to the test audience and the panel shows that the test is running, along with a countdown to when Mautic picks the winner.

> After the wait time passes, the ``mautic:email:sendwinner`` command evaluates each variant against your winner criteria, marks the best performer with a **Winner** badge in the panel, and sends the remaining Emails using the winning variant's content through the standard broadcast command. For setup details, see the :ref:`Determine A/B test winner <determine ab test winner>` section in :doc:`/configuration/cron_jobs`.

Reviewer: I'm not sure if we should mention any command here. I don't think general user needs to understand the command or how it works under the hood.

.. vale off

Viewing A/B test results
------------------------

.. vale on

> Once the test is sending, the panel shows an **A/B Test Stats** button. Select it to compare how each variant is performing and to see which variant Mautic picked as the winner.

Reviewer: Add this image:

<img width="2067" height="1081" alt="Image" src="https://github.com/user-attachments/assets/582c4c4a-09e7-4ad0-aa7c-799108ded0dd" />

- Name it: ab_test_stats.png
- Add descriptive alt text. Do not start with "screenshot of" or anything alike.

.. vale on

.. vale off

#. Open a Segment Email that you haven't sent yet.
#. In the right-hand column of the Email detail view, find the **Convert to A/B test** card and select it.

   |

   .. image:: images/emails/email_ab_test_convert_card.png
      :width: 800
      :alt: The 'Convert to A/B test' card in the right-hand column of a Segment Email.

   |

#. In the **Convert to A/B test email** dialog, set the initial test parameters:

   * **How to pick the winner** - the criteria Mautic uses to choose the winning variant, such as **Read rate** or **Clickthrough rate**.
   * **Wait before picking winner** - how long Mautic waits - in hours - after sending the test Emails before it picks the winner. This gives Contacts time to open and click. The default is 24 hours.
   * **Test audience size** - the percentage of your Contacts that receive the test Emails, split evenly between variants. The remaining Contacts receive the winning version. The default is 10%.

     |

     .. image:: images/emails/email_ab_test_convert_dialog.png
        :alt: The 'Convert to A/B test email' dialog with the winner criteria, wait time, and test audience size fields.

     |

> #. Select **Save & Close**.

Reviewer: ```suggestion
#. Click **Save & Close**.
```

.. vale on

These settings only set up the test. You can change them any time before the test starts sending. After conversion, the right-hand column replaces the card with the **A/B Test** panel, where you manage variants, settings, and sending from one place.

.. vale off

Managing variants
-----------------

.. vale on

You manage the whole test from the **A/B Test** panel in the right-hand column. The original Email counts as the first variant, so you need at least one more - two in total - before you can schedule the test. Until you add one, the panel shows a reminder.

> * To add a variant:

Reviewer: ```suggestion
Adding a variant
~~~~~~~~~~~~~~~~

To add a variant:
```

  #. Select **Add variant +**. Mautic creates a copy that you can edit.
  #. Change the subject line, content, or design you want to test.
  #. Click **Save & Close**.

  Each variant appears in the panel tagged as an 'A/B variant' of the original Email. In the list of Emails, a variant has a symbol. When you hover over the symbol, it shows 'Is A/B variant'.

  |

  .. image:: images/emails/is_ab_variant_symbol.png
     :alt: The A/B variant symbol beside an Email in the Emails list, showing the 'Is A/B variant' label on hover.

  |

> * There are two ways to edit and remove a variant:

Reviewer: ```suggestion
Editing and deleting a variant
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

There are two ways to edit and delete a variant:
```

>   #. From individual variant:

Reviewer: ```suggestion
  #. From individual Email variant:
```

     #. Open the Email variant you want to modify.
     #. To edit a variant, click the **Edit** button at the top right.

>      #. To remove a variant, click the down arrow button to open the **Options**, then select **Delete**.

Reviewer: ```suggestion
     #. To edit it, click the **Edit** button at the top right.
     #. To remove it, click the down arrow button to open the **Options**, then select **Delete**.
```
